src/main/groovy/de/dkfz/roddy/execution/cluster/pbs/PBSCommand.groovy

```diff
@@ -147,6 +147,10 @@ class PBSCommand extends Command {
 
         StringBuilder qsubCall = new StringBuilder(EMPTY)
 
+        if (job.getToolScript()) {
```

Comment:
If you use both the piped script and the tool, the call will fail! Script and tool are either or. For this case, we should decide whether to throw an Exception or what is priorized. However, the default BEJob checks for xor and throws an Exception if both values are set. So you could ignore it here.